R/RunAnalyses.R

```diff
@@ -892,100 +920,118 @@ applyTrimMatchStratify <- function(ps, arguments) {
 }
 
 doTrimMatchStratify <- function(params) {
-  ps <- getPs(params$psFile)
-  ParallelLogger::logDebug(sprintf("Performing matching etc., using %s",
-                                   params$psFile))
-  ps <- applyTrimMatchStratify(ps, params$args)
-  saveRDS(ps, params$strataFile)
+  tryCatch({
```

Comment:
Yikes, this tryCatch block covers multiple function calls, but the error message will be about the saving of the stataFile.